src/App.tsx

```diff
@@ -1,11 +1,20 @@
-import AutocompleteInput from 'components/AutocompleteInput';
-import FetchWeatherData from 'components/FetchWeatherData';
+import { useState } from 'react';
+import LocationAutocomplete from 'components/LocationAutocomplete';
+import WeatherWidget from 'components/WeatherWidget';
+import Location from 'types/location';
 
 function App() {
+  const [selectedLocation, onSelectLocation] = useState<Location | null>(null);
+  const id = 'location-select';
```

Comment:
Rename me to `LOCATION_AUTOCOMPLETE` and put me outside of `App` function but in this file